examples/notebooks/stationarity_detrending_0.ipynb

```diff
@@ -0,0 +1,304 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Stationarity in time-series\n",
+    "\n",
+    "Stationarity means that the statistical properties of a a time series i.e. mean, variance and covariance do not change over time. Most statistical models require the series to be stationary to make effective and precise predictions.\n",
+    "\n",
+    "We will implement two statistical tests to check the stationarity of a time series – Augmented Dickey Fuller (“ADF”) test and Kwiatkowski-Phillips-Schmidt-Shin (“KPSS”) test. We shall also implement a method to convert a non-stationary time series into stationary. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This first cell imports standard packages and sets plots to appear inline."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "import pandas as pd\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import statsmodels.api as sm"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Sunspots dataset is used. It contains Yearly (1700-2008) data on sunspots from the National Geophysical Data Center."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sunspots = sm.datasets.sunspots.load_pandas().data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Some preprocessing is carried out on the data. The \"YEAR\" column is used in creating index."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sunspots.index = pd.Index(sm.tsa.datetools.dates_from_range('1700', '2008'))\n",
+    "del sunspots[\"YEAR\"]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The data is plotted now."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sunspots.plot(figsize=(12,8))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## ADF test\n",
+    "\n",
+    "ADF test is used to determine the presence of unit root in the series, and hence helps in understand if the series is stationary or not. The null and alternate hypothesis of this test are:\n",
+    "\n",
+    "Null Hypothesis: The series has a unit root (value of a =1).\n",
+    "\n",
+    "Alternate Hypothesis: The series has no unit root.\n",
+    "\n",
+    "If the null hypothesis in not rejected, the series is non-stationary.\n",
```

Comment:
Should reword this in terms of evidence. Something like: "if we fail to reject the null hypothesis, this test may provide evidence that the series is non-stationary".

Reply:
Alright, I shall correct it.